skill-tree/src/test/java/com/RDS/skilltree/unit/EndorsementServiceTest.java

```diff
@@ -602,4 +618,108 @@ void testCreateEndorsementWithInvalidSkill() {
         // Verify that save method is not called
         verify(endorsementRepository, never()).save(any(EndorsementModel.class));
     }
+
+    @Test
+    @Disabled
+    @DisplayName("Return unauthorized given user is not a super user")
+    public void itShouldReturnUnauthorizedIfUserIsNotSuperUser() {
+        setupUpdateEndorsementTests(false);
+
+        UUID endorsementId = UUID.randomUUID();
+        String status = EndorsementStatus.APPROVED.name();
+
+        InsufficientAuthenticationException exception =
+                assertThrows(
+                        InsufficientAuthenticationException.class,
+                        () -> endorsementService.updateEndorsementStatus(endorsementId.toString(), status));
```

Comment:
Hi @Atifsid 
According to the PR #116 , shouldn't it return AccessDeniedException exception instead of InsufficientAuthenticationException?

Reply:
It's correct here, changes have been made in #116 via [this commit](https://github.com/Real-Dev-Squad/skill-tree-backend/pull/116/commits/e6c249dc7681f47bef059d679c9e09885c648a32)